client/css/style.css

```diff
@@ -387,7 +387,7 @@ button, .user {
 	content: "\f013";
 }
 #footer .sign-out:before {
-	content: "\f13e";
+	content: "\f08b";
```

Comment:
aah yes, I remember now - the current icon is a lock which doesn't really indicate "exit" - this should be considered as a separate issue so I'll revert the icon shortly